.github/workflows/ext.yml

```diff
@@ -80,17 +80,28 @@ jobs:
       run: ./configure --enable-sockets --enable-mysqlnd --enable-http2 &&
         make clean && make -j$(sysctl -n hw.ncpu)
     - name: build5
-      run: ./configure --enable-sockets --enable-mysqlnd --enable-http2 --enable-openssl 
+      run: ./configure --enable-sockets --enable-mysqlnd --enable-http2 --enable-openssl
         --with-openssl-dir=/usr/local/opt/openssl@1.1 &&
         make clean && make -j$(sysctl -n hw.ncpu)
     - name: build6
-      run: ./configure --enable-sockets --enable-mysqlnd --enable-http2 --enable-swoole-curl --enable-swoole-json 
+      run: ./configure --enable-sockets --enable-mysqlnd --enable-http2 --enable-swoole-curl --enable-swoole-json
         --enable-openssl --with-openssl-dir=/usr/local/opt/openssl@1.1 --enable-debug-log &&
         make clean && make -j$(sysctl -n hw.ncpu)
 
   build-alpine-latest:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-version: [ '7.3', '7.4' ]
+        alpine-version: [ '3.9', '3.10', '3.11', '3.12' ]
```

Comment:
Latest one is 3.13 and it's more then x86-64 arches there

Help on enabling more tests are welcome https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/20327